src/include/internal/daos/checksum.h

```diff
@@ -13,35 +13,36 @@
 
 #include <daos/multihash.h>
 
-#define	CSUM_NO_CHUNK -1
+#define CSUM_NO_CHUNK -1
 
 /*
  * Tag used in debug logs to easily find important checksum info, making it
  * easier to trace checksums through the log files
  */
-#define CSTAG "[CSUM]"
+#define CSTAG         "[CSUM]"
 
-#define DF_C_IOD CSTAG"IOD {akey: "DF_KEY", type: %s, nr: %d, size: %lu} CSUM"
-#define DP_C_IOD(i) DP_KEY(&(i)->iod_name), \
-	(i)->iod_type == DAOS_IOD_SINGLE ? "SINGLE" : \
-	(i)->iod_type == DAOS_IOD_ARRAY ? "ARRAY" : "UNKNOWN", \
-	(i)->iod_nr, (i)->iod_size
+#define DF_C_IOD      CSTAG "IOD {akey: " DF_KEY ", type: %s, nr: %d, size: %lu} CSUM"
+#define DP_C_IOD(i)                                                                                \
+	DP_KEY(&(i)->iod_name),                                                                    \
+	    (i)->iod_type == DAOS_IOD_SINGLE  ? "SINGLE"                                           \
+	    : (i)->iod_type == DAOS_IOD_ARRAY ? "ARRAY"                                            \
+					      : "UNKNOWN",                                         \
+	    (i)->iod_nr, (i)->iod_size
 
-#define DF_C_UOID_DKEY CSTAG"OBJ ("DF_UOID", "DF_KEY")"
+#define DF_C_UOID_DKEY            CSTAG "OBJ (" DF_UOID ", " DF_KEY ")"
 #define DP_C_UOID_DKEY(oid, dkey) DP_UOID(oid), DP_KEY(dkey)
 
-#define DF_C_OID_DKEY CSTAG"OBJ ("DF_OID", "DF_KEY")"
-#define DP_C_OID_DKEY(oid, dkey) DP_OID(oid), DP_KEY(dkey)
-
-#define DF_LAYOUT "{bytes: %lu, nr: %d, even_dist: %s, cell_align: %s}"
-#define DP_LAYOUT(l) (l).cs_bytes, (l).cs_nr, DP_BOOL((l).cs_even_dist), \
-			DP_BOOL((l).cs_cell_align)
-#define	DF_CI_BUF "%"PRIu64
-#define	DP_CI_BUF(buf, len) ci_buf2uint64(buf, len)
-#define	DF_CI "{nr: %d, len: %d, first_csum: %lu, csum_buf_len: %d}"
-#define	DP_CI(ci) (ci).cs_nr, (ci).cs_len, ci2csum(ci), (ci).cs_buf_len
-#define DF_RANGE "{lo: %lu, hi: %lu, nr: %lu}"
-#define DP_RANGE(r) (r).dcr_lo, (r).dcr_hi, (r).dcr_nr
+#define DF_C_OID_DKEY             CSTAG "OBJ (" DF_OID ", " DF_KEY ")"
+#define DP_C_OID_DKEY(oid, dkey)  DP_OID(oid), DP_KEY(dkey)
+
+#define DF_LAYOUT                 "{bytes: %lu, nr: %d, even_dist: %s, cell_align: %s}"
+#define DP_LAYOUT(l)              (l).cs_bytes, (l).cs_nr, DP_BOOL((l).cs_even_dist), DP_BOOL((l).cs_cell_align)
```

Comment:
(style)  line over 100 characters
(style)  Macros with complex values should be enclosed in parentheses